### DISCUSSION TITLE
How can I open up a gap immediately when I split a container in i3?

### DISCUSSION THREAD

**Discussion by otakutyrant** [i3/i3#5546] [Q&A]
What gap I said is not that gap between containers or screen edges. If I have `bindsym $mod+v split vertical` in config and I press `mod+v`, the current container will be "splited" already, but the correspond window still occupy the whole container, unless you launch a new windows then you see two windows occupy the whole container vertically. What I want is when I press `mod+v`, the container is splited into a windows and a gap imediattely, then when a new windows launch, the gap will be replaced by the new window.

So how can I do that?

I noticed that the titlebar will be marked that it is splited after spliting. However with this feature we do not need the titlebar because a gap is obvious enough.

**Comment by okraits** [comment]
You can create a placeholder container with `i3-msg open`. That's not an official feature, though, just for testing.

**Reply by otakutyrant** [reply]
Unfortuanatly, it woundn't be replaced by new windows.
